stories/button.js

```diff
@@ -7,6 +7,7 @@ import { withKnobs, boolean, select } from '@storybook/addon-knobs/react';
 import { IconAddMediumOutline, IconAddSmallOutline } from '@teamleader/ui-icons';
 import { Button } from '../components';
 
+const elements = ['a', 'button', 'div', 'span'];
```

Comment:
I'd prefer to remove the `div` and `span` examples from the elements array as this isn't something we should be doing anyway for accessibility reasons.